docs/UserGuide.md

```diff
@@ -291,152 +350,274 @@ Format: `tag INDEX t/TAGNAME [f/]`
+    * If the Task number is 1, enter the command `mark-task 1`.
 
 ### Un-mark a task  : `unmark-task` 
 
-Format: `unmark-task TASKINDEX`
+Use the unmark-task command to mark one or more `Task` as incomplete.
 
-* Marks a `Task` as not completed.
-* The index correspond to the index of the task when in the [list-tasks](#listing-all-tasks--list-tasks) view.
-* The index **must be a positive integers** 1, 2, 3, …​.
+Format: `unmark-task TASK_NUMBER [ADDITIONAL_TASK_NUMBER]`
+
+To mark a `task` as incomplete, enter the command [list-tasks](#listing-all-tasks--list-tasks) to identify the Task number(s) of task(s) you wish to mark <br>
+Following that, type `unmark-task` followed by the Task number(s).
+
+* The `TASK_INDEX` **must be a positive integer** 1, 2, 3, …​.
+
+Examples:
+* To mark the completed task with the description `Finalise itinery` as incomplete:
+    * First, identify the Task number of the task.
+    * If the Task number is 1, enter the command `unmark-task 1`.
+
+### Delete a Task : `delete-task`
+
+Use the delete-task command to delete a specific `Task`.
+
+Format: `delete-task TASK_NUMBER`
+
+To delete a `task`, enter the command [list-tasks](#listing-all-tasks--list-tasks) to identify the Task number in the `task` list. <br>
+Following that, type "delete-task" and the task number of the task that you wish to delete.
+
+* The `TASK_NUMBER` **must be a positive integer** 1, 2, 3, …​.
+
+Examples:
+* To delete a task with the description `Buy cake`:
+    * First, identify the Task number by entering the command: `list-tasks`
+    * If the task number is 1, enter the command: `delete-task 1`
 
 ## Vendor Features
 
 ### Assigning a Vendor : `assign-vendor`
-Format: `assign-venor PERSONINDEX`
 
-* Assigns an existing `Person` at the specified `Index` to become a `Vendor`.
-* The `Vendor` now can have `Tasks` assigned to it.
-* The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​.
+Use the assign-vendor command to designate a contact as a vendor.
+
+Format: `assign-vendor PERSON_NUMBER`
+
+To designate a contact as a vendor, type "assign-vendor" followed by the Person's positional number.
+
+* The `PERSON_NUMBER` **must be a positive integer** 1, 2, 3, …​.
+* The `Vendor` can have `Tasks` assigned to them.
+
+
+Examples:
+* To designate the first contact as a vendor, enter the command:
+    * `assign-vendor 1`.
 
 ### Unassigning a Vendor : `unassign-vendor`
-Format: `unassign-venor PERSONINDEX`
 
-* Unassigns a `Vendor` at the specified `Index` such that the `Person` is no longer a `Vendor`. `Tasks` can no longer be assigned to this `Person`. 
-* The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​.
+Use the unassign-vendor command to remove a vendor designation from a contact, making them a regular contact in WedLinker.
+
+Format: `unassign-venor PERSON_NUMBER`
```

Comment:
Unassign vendor also has force functionality that checks if the vendor still has tasks assigned to them, can maybe add this in the description

Reply:
oh, I will add that in